api/src/main/resources/notebooks.yaml

```diff
@@ -854,7 +860,7 @@ definitions:
       masterDiskSize:
         type: integer
         description: |
-          Optional, the size of the disk on the master node. Minimum size is 100GB. If unspecified, default size is 500GB.
+          Optional, the size in megabytes of the disk on the master node. Minimum size is 10GB. If unspecified, default size is 100GB.
```

Comment:
Have we changed to only ask for 10 GB or 25 GB yet? Should we?

Reply:
It was just closed: https://github.com/DataBiosphere/leonardo/issues/388, not sure whether the docs were updated.